src/Platforms/OraclePlatform.php

```diff
@@ -407,28 +407,28 @@ protected function _getCreateTableSQL($name, array $columns, array $options = []
     {
         $indexes            = $options['indexes'] ?? [];
         $options['indexes'] = [];
-        $sql                = parent::_getCreateTableSQL($name, $columns, $options);
+        $sql                = [];
```

Comment:
Let's leave the parent call up here.

```suggestion
        $sql                = [parent::_getCreateTableSQL($name, $columns, $options)];
```